public/styles/components/_login.scss

```
@@ -0,0 +1,206 @@
+  }
+  
+  &__container {
+    &--intro,
+    &--form {
+      align-items: center;
+      display: flex;
+      flex-direction: column;
+      height: 100%;
+      justify-content: center;
+      z-index: 2;
+
+      p,
+      span,
+      h1,
+      label {
+        color: $white;
+      }
+    }
+  }
+
+  &__slogan {
+    margin: 40px 0 0;
+    font-size: 15px;
+    font-weight: $semi-bold;
+
+    @include desktop {
+      font-size: 20px;
+    }
+  }
+
+  &__logo {
+    @include tablet {
+      display: none;
+    }
+  }
+
+  &__head {
+    margin: 0 0 35px;
+    color: $orange;
+    text-transform: uppercase;
+
+    &::after {
+      background: {
+        color: $white;
+      }
+
+      content: '';
+      display: block;
+      width: 100%;
+      max-width: 160px;
+      height: 3px;
+    }
+  }
+
+  &__catchphrase {
+    margin: 0 0 35px;
+    font-weight: $semi-bold;
+    font-size: 20px;
+  }
+
+  &__form {
+    display: flex;
+    flex-direction: column;
+    max-width: 440px;
+    width: 100%;
+
+    @include desktop {
+      align-items: center;
+    }
+
+    label {
+      font-size: 10px;
+      font-weight: $semi-bold;
+    }
+  }
+
+  &__username,
+  &__password {
+    display: flex;
+    flex-direction: column;
+
+    @include desktop {
+      width: 100%;
+    }
+
+    label {
+      margin: 0 0 5px;
+      font-size: 15px;
+    }
+
+    input {
+      padding: 10px 5px;
+      outline: none;
+
+      &:focus {
+        border: 2px solid $orange;
+      }
+    }
+  }
+
+  &__username {
+    margin: 0 0 10px;
+
+    @include desktop {
+      margin: 0 0 20px;
+    }
+  }
+
+  &__password {
+    @include desktop {
+      margin: 0 0 25px;
+    }
+
+    input {
+      margin: 0 0 5px;
+    }
+
+    p {
+      cursor: pointer;
+      font-size: 13px;
+      font-weight: $semi-bold;
+      max-width: 110px;
+
+      @include desktop {
+        margin: 0 0 0 auto;
+      }
+    }
+  }
+
+  &__submit {
+    align-items: flex-end;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+
+    @include desktop {
+      align-items: center;
+      flex-direction: column;
+      width: 100%;
+    }
+  }
+
+  
+  &__submit-button {
+    cursor: pointer;
+    font-size: 13px;
+    font-weight: $semi-bold;
+    transition: 0.3s ease;
+
+    span {
+      color: $orange;
+    }
+
+    &:hover {
+      span {
+        color: $dark-orange;
+      }
+    }
+
+    @include desktop {
+      order: 2;
+    }
+  }
+
+  &__button {
+    background: {
+      color: $orange;
+    }
+
+    border: none;
+    cursor: pointer;
+    font-family: $raleway;
+    font-weight: $semi-bold;
+    max-width: 70px;
+    outline: none;
+    padding: 15px;
+    text-align: center;
+    text-transform: uppercase;
+    transition: 0.3s ease;
+
+    &:hover {
+      background: {
+        color: $dark-orange;
+      }
+    }
+
+    @include desktop {
+      margin: 0 0 10px;
+      order: 1;
+      max-width: unset;
```

```suggestion
      max-width: unset;
      order: 1;
```

Ordering 😂 
On another note, perhaps we could install some linter so we don't have to watch out for these.

---

Perhaps OwO